kani-compiler/src/codegen_cprover_gotoc/codegen/statement.rs

```diff
@@ -189,17 +196,11 @@ impl<'tcx> GotocCtx<'tcx> {
                             &msg_str,
                             loc,
                         ),
-                        Stmt::goto(self.current_fn().find_label(target), loc),
+                        Stmt::goto(bb_label(*target), loc),
                     ],
                     loc,
                 )
             }
-            TerminatorKind::FalseEdge { .. } | TerminatorKind::FalseUnwind { .. } => {
-                unreachable!("drop elaboration removes these TerminatorKind")
-            }
-            TerminatorKind::Yield { .. } | TerminatorKind::CoroutineDrop => {
-                unreachable!("we should not hit these cases") // why?
```

Comment:
What happened with these? Did they get removed?

Reply:
Yes. They are not included in StableMIR at least for now, since we only support the optimized MIR version, and these shouldn't occur.